mu-plugins/blocks/screenshot-preview/src/screenshot.js

```diff
@@ -105,7 +105,11 @@ function ScreenShotImg( { queryString, src, isReady = false } ) {
 	}
 
 	if ( isLoading ) {
-		return <div className="wporg-screenshot wporg-screenshot__loading">{ __( 'Loading …', 'wporg' ) }</div>;
+		return (
+			<div className="wporg-screenshot wporg-screenshot-loader">
+				<span />
```

Comment:
Could you use a pseudo element instead of a `span`?

Reply:
Done: https://github.com/WordPress/wporg-mu-plugins/pull/292/commits/c5b7f9af5dfd243da11dbede2de53375ec2844de